test_suite/cloud/test_aws.py

```diff
@@ -317,8 +317,17 @@ def test_required_packages_are_installed(self, host):
     def test_rhui_pkg_is_installed(self, host):
         with host.sudo():
             # CLOUDX-590
-            assert host.run_test('rm -f /var/lib/rpm/__db.*'), 'Failed to remove the RPM db files'
-            assert host.run_test('rpm --rebuilddb'), 'Failed to rebuild the RPM db files'
+            try:
+                assert host.run_test('rm -f /var/lib/rpm/__db.*'), 'Failed to remove the RPM db files'
+                assert host.run_test('rpm --rebuilddb'), 'Failed to rebuild the RPM db files'
+            except AssertionError:
+                workaround_error = 'chmod 755 /var/lock /var/lock/rpm ' \
```

Comment:
I suggest renaming this var to `change_permissions_cmd`.

Reply:
ACK